<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtherAmount</class>
 <widget class="QDialog" name="OtherAmount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
		background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(51, 51, 51, 255), stop:				0.525822 rgba(95, 95, 95, 255), stop:1 rgba(58, 58, 58, 255));
}

QFrame#PinFrame{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.431818, x2:1, y2:0.409364, stop:0.217877 rgba(43, 43, 43, 255), stop:0.50838 rgba(64, 64, 64, 255), stop:0.938547 rgba(37, 37, 37, 255));
border: 4px ridge #383838; 
border-radius: 10px;
}

QFrame#frame {
	background-color: rgb(75, 75, 75);	
	border: 2px inset #4a8390;
	border-radius: 20px;
}

QLabel {
	background: transparent;
	border: none;
}

QPushButton {
	font: 290 16pt &quot;Microsoft YaHei Light&quot;;	
	color: #dadada;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.431818, x2:1, y2:0.409364, stop:0.217877 			rgba(43, 	43, 43, 255), stop:0.50838 rgba(64, 64, 64, 255), stop:0.938547 rgba(37, 37, 37, 255));
	border: 2px groove #4a8390;
	border-radius: 10px;
	padding: 6px;
}

QPushButton:hover {
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0.488, y2:0, stop:0 rgba(37, 37, 37, 255), 	stop:1 rgba(104, 104, 104, 255));
}

QPushButton:pressed {
	background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0.488, y2:0, stop:0 rgba(22, 22, 22, 255), stop:1 rgba(64, 64, 64, 255));
}

QPushButton#BCancel_2 {
	color: qlineargradient(spread:reflect, x1:0, y1:0, x2:0.55, y2:0, stop:0 rgba(145, 45, 47, 255), stop:1 rgba(163, 68, 74, 255));	
}

QPushButton#BEnter_2{
	color: qlineargradient(spread:pad, x1:0, y1:0.391545, x2:1, y2:0.392, stop:0.223464 rgba(51, 182, 97, 255), stop:1 rgba(53, 251, 122, 255));
}

QLabel {		
 color: #dadada;
	font: 290 16pt &quot;Microsoft YaHei Light&quot;;

}
QLabel#label_2 {	
	
 color: #dadada;
	font: 290 14pt &quot;Microsoft YaHei Light&quot;;
}

QLabel#pirkkal {	
	color: #dadada;
	font: 350 10pt &quot;Leelawadee UI Semilight&quot;;
	qproperty-geometry:rect(610 550 101 35);
}

QLineEdit {	
font: 290 16pt &quot;Microsoft YaHei Light&quot;;
	color: #dadada;
	border: 3px ridge #383838;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.431818, x2:1, y2:0.409364, stop:0.217877 rgba(43, 43, 43, 255), stop:0.50838 rgba(64, 64, 64, 255), stop:0.938547 rgba(37, 37, 37, 255));
	border-radius:8px;

}
QPushButton#euroicon {
background: transparent;
	border: none;

}


</string>
  </property>
  <widget class="QFrame" name="PinFrame">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>155</y>
     <width>191</width>
     <height>241</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QPushButton" name="A2">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="A8">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="A9">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="A4">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="A7">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="A5">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="A0">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="A3">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="A1">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="BCancel_2">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="A6">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="BEnter_2">
      <property name="font">
       <font>
        <family>Microsoft YaHei Light</family>
        <pointsize>16</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>E</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>16</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Syötä haluamasi summa</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei Light</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>ja klikkaa Enter</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>321</width>
     <height>361</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="haluttusumma">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei Light</family>
      <pointsize>16</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>IBeamCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="euroicon">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="gifsANDicons.qrc">
      <normaloff>:/icons/icons/euro.png</normaloff>:/icons/icons/euro.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="pirkkal">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>550</y>
     <width>101</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Pirkka Pankki</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>480</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>   Takaisin</string>
   </property>
   <property name="icon">
    <iconset resource="gifsANDicons.qrc">
     <normaloff>:/icons/icons/left.png</normaloff>:/icons/icons/left.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>PinFrame</zorder>
  <zorder>label_1</zorder>
  <zorder>label_2</zorder>
  <zorder>pirkkal</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources>
  <include location="gifsANDicons.qrc"/>
 </resources>
 <connections/>
</ui>
